packages/address/src/address.ts

```diff
@@ -44,7 +48,8 @@ export default class Address extends AbstractAddress {
 
   /**
    * Returns the `bech32Address` property
-   *
+   * @deprecated
+   * The `bech32Address` is now deprecated. Hexadecimal will be the only address format from this point onward.
```

Comment:
```suggestion
   * @deprecated
   * The `bech32Address` is now deprecated. Please migrate to `toB256` as this will be the standard going forward.
```

Reply:
9d2580723